analysis/build.gradle

```diff
@@ -68,7 +70,7 @@ allprojects {
     apply plugin: "org.jlleitschuh.gradle.ktlint"
 
     jacoco {
-        toolVersion = jacoco_version
+      toolVersion = jacoco_version
```

Comment:
Why have you changed the spaces here?

Reply:
That's weird. Will fix it in the next commit